### ISSUE TITLE
[Gallery] [Firefox] Uno logo and icons is not displaying correctly

### ISSUE THREAD

**Issue by Jadhielv** [unoplatform/uno#6913]
## Current behavior:
![New Issue](https://user-images.githubusercontent.com/24376900/131387334-694e8dd4-4e31-435b-8310-ff1183540bbd.gif)

## Expected behavior:
![imagen](https://user-images.githubusercontent.com/24376900/131389835-43b49109-fefb-4c32-93c6-1f6fe80996ea.png)

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [x] Windows
- [ ] Build tasks
- [ ] Solution Templates

**Comment by Youssef1313** [comment]
I'm not able to reproduce:

![image](https://user-images.githubusercontent.com/31348972/131397561-9418e78a-6b57-4ad7-8297-f94464b62f7f.png)

**Comment by jeromelaban** [comment]
@Jadhielv it would look like the resource path is incorrect. Is there a 404 or similar error in your network tab in the browser ?

**Comment by Jadhielv** [comment]
yep @jeromelaban 

![image](https://user-images.githubusercontent.com/24376900/131403823-5c3b47bc-4a70-42f9-bf29-95067b6ab5ce.png)

![image](https://user-images.githubusercontent.com/24376900/131404028-bc9290f2-724a-4ad7-8ec3-a986be3ca0fb.png)

![image](https://user-images.githubusercontent.com/24376900/131404062-24870c63-034a-4507-bca4-5f023d1b0922.png)

**Comment by Jadhielv** [comment]
> 
> 
> I'm not able to reproduce:
> 
> ![image](https://user-images.githubusercontent.com/31348972/131397561-9418e78a-6b57-4ad7-8297-f94464b62f7f.png)

That's **Microsoft Edge** not **Firefox** @Youssef1313

![image](https://user-images.githubusercontent.com/24376900/131404430-60574872-6a20-4cbe-8b20-385d5eb75e84.png)

**Comment by Youssef1313** [comment]
@Jadhielv Yeah sorry :) However, I wasn't also able to reproduce on Firefox (same version as yours):

![image](https://user-images.githubusercontent.com/31348972/131411198-21655313-9ddc-4c19-94d7-3d9e035d7989.png)

**Comment by jeromelaban** [comment]
@Jadhielv Thanks for the update! This would look like a scaling issue. What's the windows display scaling configuration for the screen you're displaying firefox on ?

**Comment by Youssef1313** [comment]
> This would look like a scaling issue.

Indeed. I could reproduce as follows:

- Open the gallery app as normal -> works correctly.
- Set zoom level in browser to (for example) 80% and refresh. Images don't show.

**Comment by Youssef1313** [comment]
@jeromelaban Please assign me :)

**Comment by Jadhielv** [comment]
> 
> 
> @Jadhielv Thanks for the update! This would look like a scaling issue. What's the windows display scaling configuration for the screen you're displaying firefox on ?

@jeromelaban **90%**

**Comment by Jadhielv** [comment]
> 
> 
> > This would look like a scaling issue.
> 
> Indeed. I could reproduce as follows:
> 
>     * Open the gallery app as normal -> works correctly.
> 
>     * Set zoom level in browser to (for example) 80% and refresh. Images don't show.

@Youssef1313 Exactly. Btw, it's the same in other **browsers**:

**Microsoft Edge**

![image](https://user-images.githubusercontent.com/24376900/131514875-2fb14416-0668-4108-8796-af6b28e84591.png)

**Google Chrome**

![image](https://user-images.githubusercontent.com/24376900/131514802-9416a62f-87f7-4f7d-ba16-b66af3869a8c.png)

**Opera**

![image](https://user-images.githubusercontent.com/24376900/131515110-69e390f0-d234-402b-9638-d4fdf0f3fdc6.png)

**Comment by Youssef1313** [comment]
@Jadhielv Yes. It affects all browsers. In case you're hitting this outside the gallery app, you can duplicate the `.scale-100` asset without the qualifier. i.e, if you have a bunch of `MyImage.scale-XXX.extentsion`, just add `MyImage.extension`. It should fix it until #6921 goes in.
